deploy.bash

```diff
@@ -83,7 +83,7 @@ fi
 
 echo "2. Install cpan packages... "
 if [[ $step =~ (^|,)2(,|$) || $step == "*" ]]; then
-	cpan_packages=(DBI DBD::Firebird Algorithm::Diff Text::Aspell SQL::Abstract Archive::Zip
+	cpan_packages=(Module::Install DBI DBD::Firebird Algorithm::Diff Text::Aspell SQL::Abstract Archive::Zip
```

Comment:
Add this link to commit message:
https://github.com/bobtfish/text-multimarkdown/issues/33